### DISCUSSION TITLE
Umbraco Tutorial/Tour - please allow users to jump ahead to different steps

### DISCUSSION THREAD

**Discussion by jacksorjacksor** [umbraco/Umbraco-CMS#14666] [Features and ideas]
Hi!

The Umbraco "Tour" tutorial is a great resource for new starters to the Umbraco world - however, I would strongly advocate in favour of allowing all steps (six at time of writing) to be accessible for users to view without the necessitation of the completion of previous steps - if I wish to view details about "6. The Media library" why do I need to complete steps 1-5 to do so?

![image](https://github.com/umbraco/Umbraco-CMS/assets/5838388/ccd293d1-2ad2-4b06-a60b-2b302c6ea347)

This becomes increasingly frustrating as I have found an issue with "2. Create document type" which might lead to that step not being completable (will write up an issue once I do more testing). This means I can not unlocked steps 3-6.

So, I beseech thee with the greatest respect - please consider allowing users to have the freedom to access the tutorial content in any order.

Cheers!
Rich

**Comment by bergmania** [comment]
Hi @jacksorjacksor.

Often you cant use the future steps if the previous is not completed because they depend on something created in that step..

E.g you can't do (3) create content if you didn't create the doc type. etc.

**Comment by jacksorjacksor** [comment]
Sure - as I'm working through them I'm realised how tightly bound the steps are to one another.
